### ISSUE TITLE
colors in emacs 28 running in alacritty look wrong/different from emacs 27

### ISSUE THREAD

**Issue by quite** [alacritty/alacritty#5995]
I filed this bug with emacs here: https://lists.gnu.org/archive/html/bug-gnu-emacs/2022-04/msg00232.html -- but perhaps it's more likely to catch some attention and getting it reproduced here... Though I of course realise that the issue *might* be in emacs 28, or alacritty interop.

I'm trying out emacs 28, coming from 27.2 on an Arch system. I have an issue with the colors displayed when running in alacritty terminal. Running in urxvt and xterm is fine.

I see different hues of most colors, and list-colors-display looks completely different in alacritty vs urxvt. I'm attaching a screenshot of what it looks like. On the left is alacritty, on the right urxvt. Running emacs -nw -q --no-site-file -nsl

In #emacs on libera I was pointed at https://debbugs.gnu.org/cgi/bugreport.cgi?bug=48439 and I tried removing the alacritty alias, but that didn't help.

My system has ncurses 6.3 which does have terminfo for alacritty.

I should add that I tried running alacritty with default config, and also tried with alacritty compiled from git as of today.

### System

OS: Linux
Version: alacritty 0.10.1 (2844606d), and as well compiled from git today
Linux/BSD: X11, i3

**Comment by kchibisov** [comment]
You're might be using truecolors now or indexed colors? Could try `TERM=xterm-256colors emacs` and some other terminfos, like the one from urxvt.

**Comment by quite** [comment]
I see no different when trying such TERM settings.

I've played around a bit more, and I'm now wondering if it is perhaps that the combination of emacs 28 and alacritty now is doing things correctly. And it is urxvt and xterm that are flawed. I've never really used for example gnome-terminal (vte based), but I tried it now and I think the display is identical (including list-colors-display showing named colors, rather than just color-x).
